sentry_sdk/consts.py

```diff
@@ -386,6 +386,32 @@ class SPANDATA:
     """
 
 
+class SPANSTATUS:
```

Comment:
How about we make this an [enum](https://docs.python.org/3/library/enum.html#enum.Enum)?